config/jest/jest.setup.js

```diff
@@ -1,26 +1,9 @@
-const glob = require("glob");
-const { PACKAGE_NAME_VEST } = require("../../scripts/constants");
-const { packagePath, packageJson } = require("../../util");
+const glob = require('glob');
 
-const { version } = packageJson(PACKAGE_NAME_VEST);
+const { packagePath } = require('../../util');
 
-global.vestDistVersions = [];
-
-const isWatchMode = (process.argv || []).some(
-  (arg) => arg && arg.includes("--watch")
-);
-
-const VEST_DIST_FILES = glob.sync(
-  packagePath(PACKAGE_NAME_VEST, "dist", "*.js")
+global.isWatchMode = (process.argv || []).some(
```

Comment:
If I'm not wrong process.argv can't be nil (null or undefined). Isn't? Even I guess it won't be empty array either.